pkgs/test/lib/src/runner/executable_settings.dart

```diff
@@ -36,6 +36,9 @@ class ExecutableSettings {
 
   /// The path to the executable for the current operating system.
   String get executable {
+    final envVariable = Platform.environment['CHROME_EXECUTABLE'];
```

Comment:
We probably should not have put a chrome specific override in a general purpose class...

I'm pretty sure this environment variable is overriding _all_ browsers now.

Reply:
lol, whoops 😬

Reply:
https://github.com/dart-lang/test/pull/1970